shell/common/display.h

```diff
@@ -0,0 +1,47 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_SHELL_COMMON_DISPLAY_H_
+#define FLUTTER_SHELL_COMMON_DISPLAY_H_
+
+namespace flutter {
+
+/// Unique IO per display that is stable until the Flutter application restarts.
+/// See also: `flutter::Display`
+typedef uint64_t DisplayId;
+
+/// Some devices do not support multiple displays, in such cases the embedding
+/// might not provide a device identifier. This value is used as the device id
+/// in such cases. There must only be one screen in such cases.
+static constexpr DisplayId kSingleScreenDeviceId = 0;
+
+/// To be used when the display refresh rate is unknown. This is in frames per
+/// second.
+static constexpr double kUnknownDisplayRefreshRate = 0;
+
+/// Display refers to a graphics hardware system consisting of a framebuffer,
+/// typically a monitor or a screen. This class holds the various display
+/// settings.
+class Display {
```

Comment:
Like all of these terms, the term display is overloaded with terms in X11 and other platforms.  I'd prefer "screen", as that seems to be slightly less encumbered, but "display" isn't horrible, just more confusing in some contexts.

Reply:
All of the platforms that i've looked at seem to refer the screens as displays (sometime even `VirtualDisplay` like on Android). I am personally not tied to either Screen or Display, VirtualScreen for example, makes as much sense as VirtualDisplay for me.

I'd prefer to not rename this across all these files unless theres a good case for it 😊

Reply:
OK, Display it is then. :-)